lib/src/main/java/space/taran/arklib/Lib.kt

```diff
@@ -20,11 +26,43 @@ data class LinkData(
     }
 }
 
+@Parcelize
 data class ResourceId(
     val dataSize: Long,
     val crc32: Long
-) {
+) : Parcelable {
+
+    override fun toString() =
+        dataSize.toString()
+            .plus(KEY_VALUE_SEPARATOR)
+            .plus(crc32)
+
+    fun fromString(str: String): ResourceId {
+        val parts = str.split(KEY_VALUE_SEPARATOR)
+            return ResourceId(
+                parts[0].toLong(),
+                parts[1].toLong()
+       )
+    }
+
     companion object {
+
+        const val KEY_VALUE_SEPARATOR = "-"
+
+        class Converter {
+            @TypeConverter
+            fun fromString(str: String): ResourceId {
+                val idAry = str.split(KEY_VALUE_SEPARATOR)
+                return create(
```

Comment:
I've added `fromString` to the class itself (useful for routine stuff, like reading id from tag storage).

Do we need this method or can we just wire this method to new method?